feature-libs/cart/quick-order/components/cart-quick-order-form/cart-quick-order-form.component.ts

```diff
@@ -141,6 +149,13 @@ export class CartQuickOrderFormComponent implements OnInit, OnDestroy {
     );
   }
 
+  /**
+   * @deprecate since SPA_2406
+   *
+   * This method is no longer needed since BadRequestHandler.handleUnknownIdentifierError was introduced.
+   * If this method is used an unnecessary duplicated error message will appear in the UI.
+   * Therefore this method will be removed.
```

Comment:
```suggestion
   * @deprecated since 2211.24
   *
   * This method is no longer needed since BadRequestHandler.handleUnknownIdentifierError was introduced.
   * If this method is used an unnecessary duplicated error message will appear in the UI.
   * Therefore this method will be removed.
   *
   * You can enable the Feature Toggle 'cartQuickOrderRemoveListenToFailEvent'
   * to stop calling this method by default.
```